SystemServices/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.7.2] - 2023-10-12
```

Comment:
Pls increment major version (2.0.0) since we are removing API. You would FYI refer to https://github.com/rdkcentral/RDKServices#versioning